flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -311,7 +304,7 @@ class FlxTilemapExt extends FlxTilemap
 	
 	/**
 	 * THIS IS A COPY FROM FlxTilemap
-	 * I've only swapped lines 386 and 387 to give DrawTilemap() a chance to set the buffer dirty
```

Comment:
Hm, yeah, referencing the exact lines here is a bad idea since that's bound to change...

Maybe there should be a comment directly before the mentioned change.